src/arm/bbai-bone-buses.dtsi

```diff
@@ -0,0 +1,457 @@
+		};
+
+		led_P8_36 {
+			gpios = <&gpio8 10 0>;
+			status = "reserved";
+		};
+
+		led_P8_37 {
+			gpios = <&gpio8 8 0>;
+			status = "reserved";
+		};
+
+		led_P8_38 {
+			gpios = <&gpio8 9 0>;
+			status = "reserved";
+		};
+
+		led_P8_39 {
+			gpios = <&gpio8 6 0>;
+			status = "reserved";
+		};
+
+		led_P8_40 {
+			gpios = <&gpio8 7 0>;
+			status = "reserved";
+		};
+
+		led_P8_41 {
+			gpios = <&gpio8 4 0>;
+			status = "reserved";
+		};
+
+		led_P8_42 {
+			gpios = <&gpio8 5 0>;
+			status = "reserved";
+		};
+
+		led_P8_43 {
+			gpios = <&gpio8 2 0>;
+			status = "reserved";
+		};
+
+		led_P8_44 {
+			gpios = <&gpio8 3 0>;
+			status = "reserved";
+		};
+
+		led_P8_45 {
+			gpios = <&gpio8 0 0>;
+			status = "reserved";
+		};
+
+		led_P8_46 {
+			gpios = <&gpio8 1 0>;
+			status = "reserved";
+		};
+
+		led_P9_11 {
+			gpios = <&gpio8 17 0>;
+			status = "reserved";
+		};
+
+		led_P9_12 {
+			gpios = <&gpio5 0 0>;
+			status = "reserved";
+		};
+
+		led_P9_13 {
+			gpios = <&gpio6 12 0>;
+			status = "reserved";
+		};
+
+		led_P9_14 {
+			gpios = <&gpio4 25 0>;
+			status = "reserved";
+		};
+
+		led_P9_15 {
+			gpios = <&gpio3 12 0>;
+			status = "reserved";
+		};
+
+		led_P9_16 {
+			gpios = <&gpio4 26 0>;
+			status = "reserved";
+		};
+
+		led_P9_17 {
+			gpios = <&gpio7 17 0>;
+			status = "reserved";
+		};
+
+		led_P9_18 {
+			gpios = <&gpio7 16 0>;
+			status = "reserved";
+		};
+
+		led_P9_19 {
+			gpios = <&gpio7 3 0>;
+			status = "reserved";
+		};
+
+		led_P9_20 {
+			gpios = <&gpio7 4 0>;
+			status = "reserved";
+		};
+
+		led_P9_21 {
+			gpios = <&gpio3 3 0>;
+			status = "reserved";
+		};
+
+		led_P9_22 {
+			gpios = <&gpio6 19 0>;
+			status = "reserved";
+		};
+
+		led_P9_23 {
+			gpios = <&gpio7 11 0>;
+			status = "reserved";
+		};
+
+		led_P9_24 {
+			gpios = <&gpio6 15 0>;
+			status = "reserved";
+		};
+
+		led_P9_25 {
+			gpios = <&gpio6 17 0>;
+			status = "reserved";
+		};
+
+		led_P9_26 {
+			gpios = <&gpio6 14 0>;
+			status = "reserved";
+		};
+
+		led_P9_27 {
+			gpios = <&gpio4 15 0>;
+			status = "reserved";
+		};
+
+		led_P9_28 {
+			gpios = <&gpio4 17 0>;
+			status = "reserved";
+		};
+
+		led_P9_29 {
+			gpios = <&gpio5 11 0>;
+			status = "reserved";
+		};
+
+		led_P9_30 {
+			gpios = <&gpio5 12 0>;
+			status = "reserved";
+		};
+
+		led_P9_31 {
+			gpios = <&gpio5 10 0>;
+			status = "reserved";
+		};
+
+		led_P9_41 {
+			gpios = <&gpio6 20 0>;
+			status = "reserved";
+		};
+
+		led_P9_42 {
+			gpios = <&gpio4 18 0>;
+			status = "reserved";
+		};
+	};
+};
+
+
+// UARTs
+bone_uart_1: &uart10 {
+	symlink = "bone/uart/1";
+};
+
+bone_uart_2: &uart3 {
+	symlink = "bone/uart/2";
+};
+
+bone_uart_3: &ocp{
+	// not available
+};
+
+bone_uart_4: &uart5 {
+	symlink = "bone/uart/4";
+};
+
+bone_uart_5: &uart8 {
+	symlink = "bone/uart/5";
+};
+
+// I2Cs
+bone_i2c_1: &i2c5 {
+	symlink = "bone/i2c/1";
+};
+
+bone_i2c_2: &i2c4 {
+	// symlink = "bone/i2c/2";
+};
+
+bone_i2c_2a: &ocp {
+	// Not available
```

Comment:
I am not sure, but I saw no problems. I will do this instead of using &ocp:

```
&dra7_pmx_core {
	not_avaulable {

	};
};

.....

bone_i2c_2a: &not_avaulable {
	// Not available
};
```

I hope this is looks better ^